recipes/h5py/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2.6.0" %}
+
+package:
+  name: h5py
+  version: {{ version }}
+
+source:
+  fn: h5py-{{ version }}.tar.gz
+  url: https://github.com/h5py/h5py/archive/{{ version }}.tar.gz
```

Comment:
Not on PyPI yet, but has been requested. See this issue ( https://github.com/h5py/h5py/issues/669 ) for more info.